cmd_windows.go

```diff
@@ -0,0 +1,9 @@
+// +build windows
+
+package overseer
+
+import "syscall"
+
+func setSysProcAttr() *syscall.SysProcAttr {
+	return &syscall.SysProcAttr{}
```

Comment:
I don't think this line is necessary.
The people from "supervisord" are doing exactly the same thing and they don't call the function at all:
https://github.com/ochinchina/supervisord/blob/master/process/pdeathsig_windows.go

Reply:
I think that is because there is no return in their version? I need the return value to match the `SysProcAttr` interface or Go will complain I think? I could be wrong as I'm very new to Go